src/document.h

```diff
@@ -663,6 +666,7 @@ struct Document {
         if (hover.g) hover.g->DrawHover(this, dc, hover);
         if (scaledviewingmode) { dc.SetUserScale(1, 1); }
         if (scrolltoselection) {
+            Zoom(initialzoomlevel, dc);

```

Comment:
this would appear to do it on every draw, not just the first?

Reply:
`scrolltoselection` is switched to `false` after the first draw.